contrib/sip_proxy/filters/network/source/conn_manager.h

```diff
@@ -360,6 +337,12 @@ class ConnectionManager : public Network::ReadFilter,
   void sendLocalReply(MessageMetadata& metadata, const DirectResponse& response, bool end_stream);
   void doDeferredTransDestroy(ActiveTrans& trans);
   void resetAllTrans(bool local_reset);
+  absl::string_view localAddress() {
```

Comment:
Why construct a string_view for a const&? Might as well keep it const& return.

Reply:
Thanks. updated.